src/layout/DefaultLayout/ArticleLayout.tsx

```diff
@@ -1,25 +1,12 @@
-import React, { useEffect } from 'react'
+import React from 'react'
 import { GovBanner } from '@trussworks/react-uswds'
-import { useTheme } from 'next-themes'
 
 import styles from './DefaultLayout.module.scss'
 
 import Header from 'components/Header/Header'
 import Footer from 'components/Footer/Footer'
-import { useGetThemeQuery } from 'operations/portal/queries/getTheme.g'
 
 const ArticleLayout = ({ children }: { children: React.ReactNode }) => {
-  const { data } = useGetThemeQuery()
-  const { setTheme } = useTheme()
-
-  useEffect(() => {
-    if (data) setTheme(data.theme)
-  }, [data])
-
-  if (!data) {
-    return null
-  }
```

Comment:
Question: If I recall correctly we did this to avoid flashing, but I don't see handling of it else where. Did you check to see if logging in with dark mode as the preference flashed with out it?

Reply:
I've been testing this and I'm not seeing a flash, so I'm inclined to leave this out. If others are seeing a flash though, we may need to move this logic back to each of the individual layout components like we previously discussed.